### ISSUE TITLE
Focus order after accepting Terms of use

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#412]
While using voiceover, after accepting the Terms of use; the focus goes to a random article instead of going to the title of the screen which should be Publications.

**Comment by zhuelle** [comment]
Need prioritization.

**Comment by payeisa** [comment]
Priority added.

**Comment by weglmar** [comment]
@zhuelle Is this ticket still relevant since we have made modifications to the Terms of Use screen? reference: #460
